tests/serializers/tests.py

```diff
@@ -411,6 +411,187 @@ def test_indentation_whitespace(self):
             if re.search(r'.+,\s*$', line):
                 self.assertEqual(line, line.rstrip())
 
+    def test_helpful_error_message_invalid_pk(self):
+        """
+        #19820 - If there is an invalid primary key, expect that the error message
+                 tells the user what the model associated with the pk is.
+        """
+        test_string = """
+        [
+        {
+            "pk": "badpk",
+            "model": "serializers.player",
+            "fields": {
+                "name": "Bob",
+                "rank": 1,
+                "team": "Team"
+            }
+        }
+        ]
+        """
+
```

Comment:
chop blank line I would probably consolidate the opening closing brackets a bit like the other tests do so the test doesn't appear so long